src/coreclr/System.Private.CoreLib/src/System/RuntimeHandles.cs

```diff
@@ -1039,7 +1039,14 @@ internal static partial Interop.BOOL IsCAVisibleFromDecoratedType(
             QCallModule sourceModule);
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern MethodAttributes GetAttributes(RuntimeMethodHandleInternal method);
+        private static extern MethodAttributes GetAttributesInternal(RuntimeMethodHandleInternal method);
+
+        internal static MethodAttributes GetAttributes(RuntimeMethodHandleInternal method)
+        {
+            if (method.IsNullHandle())
+                throw new ArgumentNullException(SR.Arg_InvalidHandle);
```

Comment:
I went over all places that throw ArgumentNullException(SR.Arg_InvalidHandle) in this PR. All of them can be asserts instead. The callers always guarantee that the handle is non-null.

Reply:
Thanks. I checked a few and removed when I could convince myself it was non-null. There were a few that I couldn't be sure of. Will update.